extensions/cornerstone/src/handleSegmentationStorage.js

```diff
@@ -1,8 +1,8 @@
-import * as dcmjs from "dcmjs";
+import * as dcmjs from 'dcmjs';
 
-import OHIF from "@ohif/core";
-import cornerstone from "cornerstone-core";
-import cornerstoneTools from "cornerstone-tools";
+import OHIF from '@ohif/core';
```

Comment:
@dannyrb Are these expected changes from the linter? If so, can we just run the linter on everything at once so we don't see anything in the PRs anymore?

Reply:
Double quotes to single quotes is a change made by `prettier`.

1. Verify husky pre-commit is working for prettier/eslint autofix for modified files
    a. `lint-staged` helps with this
2. Apply changes to all files

_Most_ files should be linted already. Some slight rule discrepancies may have existed across repositories, but now that we have unified rules, it should allow us to be more consistent.